node/core/approval-voting/src/lib.rs

```diff
@@ -576,6 +578,7 @@ async fn handle_approved_ancestor(
 
 	use bitvec::{order::Lsb0, vec::BitVec};
 
+	let mut span = polkadot_node_jaeger::hash_span(&target, "approved-ancestor");
```

Comment:
If the span exists, it would make sense to add the `Stage` right away, mostly for consistency.